CHANGELOG.md

```diff
@@ -1,7 +1,19 @@
 # Changelog
 
 ## [0.12.5](https://github.com/brave/browser-laptop/releases/v0.12.5dev)
-- TODO
+- Added recovery and backup of Brave Wallet ([#4369](https://github.com/brave/browser-laptop/issues/4369))([#3350](https://github.com/brave/browser-laptop/issues/3350))
+- Added opt-out for crash reporting ([#4479](https://github.com/brave/browser-laptop/issues/4479))([#4606](https://github.com/brave/browser-laptop/issues/4606))
+- Added numerous UI style changes ([#4627](https://github.com/brave/browser-laptop/issues/4627))([#4624](https://github.com/brave/browser-laptop/issues/4624))([#4566](https://github.com/brave/browser-laptop/issues/4566))([#4603](https://github.com/brave/browser-laptop/issues/4603))([#4562](https://github.com/brave/browser-laptop/issues/4562))([#4616](https://github.com/brave/browser-laptop/issues/4616))([#4537](https://github.com/brave/browser-laptop/issues/4537))([#1725](https://github.com/brave/browser-laptop/issues/1725))([#4614](https://github.com/brave/browser-laptop/issues/4614))([#4658](https://github.com/brave/browser-laptop/issues/4658))([#4636](https://github.com/brave/browser-laptop/issues/4636))([#4292](https://github.com/brave/browser-laptop/issues/4292))
+- Added the ability to disable spellcheck for custom filters list ([#4637](https://github.com/brave/browser-laptop/issues/4637))([#4619](https://github.com/brave/browser-laptop/issues/4619))
```

Comment:
This one is just a bugfix and not an option. 
And one of those issues links to the PR, better to just remove that. No duplicates listed either, leave those out if there are any.